README.md

```diff
@@ -487,6 +508,9 @@ if (result.error) throw result.error;
     sourceMap: {
         // source map options
     },
+    nameCache: {
+        // name cache object
+    },
```

Comment:
Rather than putting an object literal here which is probably never the form they'd use, I'd suggest the following:
```
    nameCache: null, // or specify a name cache object
```